### PR TITLE
UXD-533 Support sort field limit

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
Support sort field limit

### Notes ✏️

- Added `isAddSortDisabled` prop on `<Sort>` component
- Added `maximumSort` parameter in `useSort` hook to support the case of limited sort fields

ezgif com-gif-maker (2)

Note: there might be changes for indicating you've reached the maximum number, follow up with @mattcrest 

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [x] MINOR (backward compatible) change to `@paprika/action-bar`
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/ActionBar/src/components/Sort/Sort.js [hunk 1]
```diff
@@ -58,6 +60,9 @@ export default function Sort(props) {
   }
 
   function handleClose() {
+    setTimeout(() => {
+      console.log(document.activeElement);
+    }, 0);
```
Comment (mikrotron) [inline packages/ActionBar/src/components/Sort/Sort.js:65]: ✂️

File: packages/ActionBar/src/components/Sort/Sort.js [hunk 2]
```diff
@@ -72,6 +77,11 @@ export default function Sort(props) {
     onCancel();
   }
 
+  function handleAddSort() {
+    fieldsRef.current.focus();
```
Comment (mikrotron) [inline packages/ActionBar/src/components/Sort/Sort.js:81]:
```suggestion
    if (fieldsRef.current) fieldsRef.current.focus();
```

File: packages/ActionBar/src/components/Sort/Sort.js (hunk 1 above)
Comment (allison-c) [reply packages/ActionBar/src/components/Sort/Sort.js:65]: 😅

File: packages/ActionBar/src/hooks/useSort/useSort.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ function sortData({ sortedFields, columns, data, locale }) {
   return sortedData;
 }
 
-export default function useSort({ data = null, columns }) {
+export default function useSort({ data = null, columns, maximumSort = null }) {
```
Comment (jamiek-acl) [inline packages/ActionBar/src/hooks/useSort/useSort.js:24]: Wondering what everyone thinks of the name `maximumSort`? Maybe `maximumSortFields` is better? `maximumSorts`? Or shorten it to `maxSorts`?

**Review by jamiek-acl — APPROVED** [review]
image

**Pull request merged**